site/_sprints/247.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Sprint 247 Details
 sprint_number: 247
-slides:
-recording:
+slides: https://www.slideshare.net/slideshow/manageiq-sprint-246-review-slide-deck-bba3/274786368
```

Comment:
wrong link?  246?

Reply:
weird - let me check

Reply:
ok, I accidentally created the slide deck and put the wrong number, then went back and edited it - so the URL stayed.

That's gross - let me delete and re-add the slide deck

Reply:
wow, I guessed it was either the wrong link or named incorrectly when it was created.  Thanks for fixing it.